homeassistant/components/upnp.py

```diff
@@ -14,15 +14,15 @@
 from homeassistant.helpers import discovery
 from homeassistant.util import get_local_ip
 
-REQUIREMENTS = ['miniupnpc==2.0.2']
+REQUIREMENTS = ['pyupnp-async==0.1.0.1']
 DEPENDENCIES = ['http']
 
 _LOGGER = logging.getLogger(__name__)
 
 DEPENDENCIES = ['api']
 DOMAIN = 'upnp'
 
-DATA_UPNP = 'UPNP'
+UPNP_DEVICE = 'upnp_device'
```

Comment:
We use normally `DATA_*` for this and that make it more readable

Reply:
I changed it because I founded unreadable, but no issues changing it back